spec.md

```diff
@@ -763,13 +756,87 @@ For success responses, the following fields are defined:
 }
 ```
 
-If the successful response includes a `state` of `failed` then the Platform
-MUST send a deprovision request to the Service Broker to prevent an orphan
-being created on the Service Broker. However, while the Platform will attempt
+If the response contains `"state": "failed"` then the Platform MUST send a
+deprovision request to the Service Broker to prevent an orphan being created on
+the Service Broker. However, while the Platform will attempt
 to send a deprovision request, Service Brokers MAY automatically delete
 any resources associated with the failed bind request on their own.
 
-### Polling Interval and Duration
+## Polling Last Operation for Service Bindings
+
+When a broker returns status code `202 Accepted` for [Binding](#binding) or
+[Unbinding](#unbinding), the Platform will begin polling the
+`/v2/service_instances/:instance_id/service_bindings/:binding_id/last_operation`
+endpoint to obtain the state of the last requested operation.
+
+Returning `"state": "succeeded"` or `"state": "failed"` will cause the Platform
+to cease polling and, in the case of a [Binding](#binding) request, information
+regarding the Service Binding can then immediately be fetched using the
+[Fetching a Service Binding](#fetching-a-service-binding) endpoint.
+
+### Request
+
+#### Route
+`GET /v2/service_instances/:instance_id/service_bindings/:binding_id/last_operation`
+
+`:instance_id` MUST be a globally unique non-empty string.
```

Comment:
Technically, this should be:
`:instance_id` is the identifier of a previously provisioned instance.
same for binding_id